src/robusta/core/sinks/pagerduty/pagerduty_sink.py

```diff
@@ -120,9 +227,39 @@ def __to_unformatted_text(block: BaseBlock) -> str:
         elif isinstance(block, KubernetesDiffBlock):
             return "\n".join(
                 map(
-                    lambda diff: f"{diff.path}: {diff.other_value} ==> {diff.value}",
+                    lambda diff: f"* {diff.formatted_path}",
                     block.diffs,
                 )
             )
 
         return ""
+
+    @staticmethod
+    def __to_unformatted_text_for_changes(block: BaseBlock) -> Optional[list[str]]:
+        if isinstance(block, KubernetesDiffBlock):
+            return list(map(
+                lambda diff: diff.formatted_path,
+                block.diffs,
+            ))
+
+        return None
+
+    # fetch the changed values from the block
+    @staticmethod
+    def __block_to_changes(block: BaseBlock, enrichment: Enrichment) -> Optional[Dict[str, Any]]:
+        if isinstance(block, KubernetesDiffBlock):
+            operation = enrichment.annotations.get("operation")
+
+            change_count = 0
+            if operation == K8sOperationType.UPDATE:
+                change_count = block.num_modifications
+            elif operation == K8sOperationType.CREATE:
```

Comment:
I think we don't need the clause for CREATE and DELETE, right?

Reply:
Yes, not required, I have removed them.